src/insights_client/__init__.py

```diff
@@ -214,16 +204,26 @@ def run_phase(phase, client, validated_eggs):
     ] + sys.argv[1:]
     config = client.get_conf()
 
-    all_eggs = [ENV_EGG, NEW_EGG] + validated_eggs
+    all_eggs = [NEW_EGG] + validated_eggs
+    if ENV_EGG is not None:
+        all_eggs = [ENV_EGG] + all_eggs
+        logger.debug("ENV_EGG defined as %s." % ENV_EGG)
+    else:
+        logger.debug("ENV_EGG not defined, skipping.")
```

Comment:
I don't think we need to log this.

Reply:
Although it’s not the most interesting piece of information, I think it still brings some value. I can imagine a case when you think you have your ENV_EGG set, but it doesn’t propagate to the actual environment the Client is run in.